src/main/java/seedu/address/logic/commands/PomCommand.java

```diff
@@ -100,13 +116,18 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(Messages.MESSAGE_INVALID_TASK_TO_BE_DONED);
         }
 
+        String resultText =
+                model.getPomodoroTask() == null
+                        ? "Pomming task: " + taskToPom.toString()
+                        : "Switched pom task: " + taskToPom.toString();
+
         // Update the pomodoro model
         model.setPomodoroTask(taskToPom);
 
         System.out.println("Hardy: " + (pm.getDefaultStartTime()));
```

Comment:
Should use logger for useful information. they actually recommend logging some stuff